lib/vast/Dialect/HighLevel/HighLevelOps.cpp

```diff
@@ -268,6 +268,58 @@ namespace vast::hl
         st.addTypes(type);
     }
 
+    mlir::ParseResult IfOp::parse(mlir::OpAsmParser &parser, mlir::OperationState &result) {
+        std::unique_ptr<::mlir::Region> condRegionRegion = std::make_unique<::mlir::Region>();
+        std::unique_ptr<::mlir::Region> thenRegionRegion = std::make_unique<::mlir::Region>();
+        std::unique_ptr<::mlir::Region> elseRegionRegion = std::make_unique<::mlir::Region>();
+
+        if (parser.parseRegion(*condRegionRegion))
+            return ::mlir::failure();
+
+        if (parser.parseKeyword("then"))
+            return mlir::failure();
```

Comment:
Inconsistency with above `::mlir`. Why is `::mlir` used here? (Same in the `unique_ptrs`).